internal/builders/docker/pkg/config.go

```diff
@@ -86,7 +86,7 @@ func NewDockerBuildConfig(io *InputOptions) (*DockerBuildConfig, error) {
 		return nil, err
 	}
 
-	if err = validatePath(io.BuildConfigPath); err != nil {
+	if err = utils.PathIsUnderCurrentDirectory(io.BuildConfigPath); err != nil {
```

Comment:
this is an early validation, and the main builder will use a safeRead anyway, correct?

Reply:
yes! it's read at the end of the file with safe read.